### PR TITLE
AST-3626 - Improvement: Improved accessibility by adding aria-label and title attributes to the shopping cart link.

### PR DESCRIPTION
**Pull request by yssambare12**
When we add a cart icon on the mobile/tablet header it provides an error Links do not have a discernible name inside the PageSpeed Insight test result 

**Steps To Reproduce:** 

1. Add a cart for mobile devices and check on https://pagespeed.web.dev/  site

**Refer to this video:** https://d.pr/v/VQ8UIO

### Types of changes

- Accessibility Improvement for screen readers.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 1]
```diff
@@ -3264,7 +3264,19 @@ public function woo_mini_cart_markup() {
...
+						$this->astra_get_cart_link();
+						$cart_link_html = ob_get_clean();
+		
+						$cart_link_html = str_replace(
+							'<a href',
+							'<a aria-label="View Shopping Cart" title="View Shopping Cart" href',
+							$cart_link_html
+						);
+		
+						echo $cart_link_html;
```
Comment (imnavanath) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php:3278]:
```suggestion
						echo do_shortcode( $cart_link_html );
```

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 2]
```diff
@@ -3307,15 +3307,19 @@ public function astra_get_cart_link() {
...
 			// Do not redirect to Cart Page in Customizer Preview & when 'Cart Page' option is not selected.
 			if ( is_customize_preview() && 'redirect' !== astra_get_option( 'woo-header-cart-click-action' ) ) {
 				$woo_cart_link = '#';
 			}
 
 			$cart_total_label_position = astra_get_option( 'woo-header-cart-icon-total-label-position' );
 			?>
-			<a href="<?php echo esc_url( $woo_cart_link ); ?>" class="cart-container ast-cart-desktop-position-<?php echo esc_attr( $cart_total_label_position['desktop'] ); ?> ast-cart-mobile-position-<?php echo esc_attr( $cart_total_label_position['mobile'] ); ?> ast-cart-tablet-position-<?php echo esc_attr( $cart_total_label_position['tablet'] ); ?> ">
+			<a href="<?php echo esc_url( $woo_cart_link ); ?>" 
+			   class="cart-container ast-cart-desktop-position-<?php echo esc_attr( $cart_total_label_position['desktop'] ); ?> ast-cart-mobile-position-<?php echo esc_attr( $cart_total_label_position['mobile'] ); ?> ast-cart-tablet-position-<?php echo esc_attr( $cart_total_label_position['tablet'] ); ?> "
```
Comment (imnavanath) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php:3321]:
```suggestion
			<a href="<?php echo esc_url( $woo_cart_link ); ?>" class="cart-container ast-cart-desktop-position-<?php echo esc_attr( $cart_total_label_position['desktop'] ); ?> ast-cart-mobile-position-<?php echo esc_attr( $cart_total_label_position['mobile'] ); ?> ast-cart-tablet-position-<?php echo esc_attr( $cart_total_label_position['tablet'] ); ?> "
```

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 3]
```diff
@@ -3307,15 +3307,18 @@ public function astra_get_cart_link() {
...
 			// Do not redirect to Cart Page in Customizer Preview & when 'Cart Page' option is not selected.
 			if ( is_customize_preview() && 'redirect' !== astra_get_option( 'woo-header-cart-click-action' ) ) {
 				$woo_cart_link = '#';
 			}
 
 			$cart_total_label_position = astra_get_option( 'woo-header-cart-icon-total-label-position' );
 			?>
-			<a href="<?php echo esc_url( $woo_cart_link ); ?>" class="cart-container ast-cart-desktop-position-<?php echo esc_attr( $cart_total_label_position['desktop'] ); ?> ast-cart-mobile-position-<?php echo esc_attr( $cart_total_label_position['mobile'] ); ?> ast-cart-tablet-position-<?php echo esc_attr( $cart_total_label_position['tablet'] ); ?> ">
+			<a href="<?php echo esc_url( $woo_cart_link ); ?>" class="cart-container ast-cart-desktop-position-<?php echo esc_attr( $cart_total_label_position['desktop'] ); ?> ast-cart-mobile-position-<?php echo esc_attr( $cart_total_label_position['mobile'] ); ?> ast-cart-tablet-position-<?php echo esc_attr( $cart_total_label_position['tablet'] ); ?> "
+			   aria-label="<?php echo $aria_label; ?>">
```
Comment (imnavanath) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php:3321]:
```suggestion
			<a href="<?php echo esc_url( $woo_cart_link ); ?>" class="cart-container ast-cart-desktop-position-<?php echo esc_attr( $cart_total_label_position['desktop'] ); ?> ast-cart-mobile-position-<?php echo esc_attr( $cart_total_label_position['mobile'] ); ?> ast-cart-tablet-position-<?php echo esc_attr( $cart_total_label_position['tablet'] ); ?>" aria-label="<?php echo $aria_label; ?>">
```

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 4]
```diff
@@ -3307,15 +3307,17 @@ public function astra_get_cart_link() {
...
+		
 			// Do not redirect to Cart Page in Customizer Preview & when 'Cart Page' option is not selected.
 			if ( is_customize_preview() && 'redirect' !== astra_get_option( 'woo-header-cart-click-action' ) ) {
 				$woo_cart_link = '#';
 			}
 
 			$cart_total_label_position = astra_get_option( 'woo-header-cart-icon-total-label-position' );
 			?>
-			<a href="<?php echo esc_url( $woo_cart_link ); ?>" class="cart-container ast-cart-desktop-position-<?php echo esc_attr( $cart_total_label_position['desktop'] ); ?> ast-cart-mobile-position-<?php echo esc_attr( $cart_total_label_position['mobile'] ); ?> ast-cart-tablet-position-<?php echo esc_attr( $cart_total_label_position['tablet'] ); ?> ">
+			<a href="<?php echo esc_url( $woo_cart_link ); ?>" class="cart-container ast-cart-desktop-position-<?php echo esc_attr( $cart_total_label_position['desktop'] ); ?> ast-cart-mobile-position-<?php echo esc_attr( $cart_total_label_position['mobile'] ); ?> ast-cart-tablet-position-<?php echo esc_attr( $cart_total_label_position['tablet'] ); ?>" aria-label="<?php echo $aria_label; ?>">
```
Comment (imnavanath) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php:3320]:
```suggestion
			<a href="<?php echo esc_url( $woo_cart_link ); ?>" class="cart-container ast-cart-desktop-position-<?php echo esc_attr( $cart_total_label_position['desktop'] ); ?> ast-cart-mobile-position-<?php echo esc_attr( $cart_total_label_position['mobile'] ); ?> ast-cart-tablet-position-<?php echo esc_attr( $cart_total_label_position['tablet'] ); ?>" aria-label="<?php echo esc_attr( $aria_label ); ?>">
```

File: changelog.txt [hunk 5]
```diff
@@ -17,6 +17,7 @@ v4.5.0
 - Fix: EDD - Mini cart widget UI broken in the RTL mode.
 - Fix: Enable Smooth Scroll to ID causes a jerk issue on the comment navigation section.
 - Fix: External JavaScript script not loading properly on frontend.
+- Improvement: Improved accessibility by adding aria-label and title attributes to the shopping cart link.
```
Comment (imnavanath) [inline changelog.txt:20]: Add changelog in latest release on top of this file

**Review by imnavanath — APPROVED** [review]

File: changelog.txt (hunk 5 above)
Comment (yssambare12) [reply changelog.txt]: Added.

**Pull request merged**